ddtrace/contrib/gunicorn/__init__.py

```diff
@@ -1,36 +1,3 @@
-"""
-**Note:** ``ddtrace-run`` and Python 2 are both not supported with `Gunicorn <https://gunicorn.org>`__.
```

Comment:
Do we still need any instructions regarding if/how to use `ddtrace-run` or manually including `ddtrace.bootstrap.sitecustomize` ?

Is including `import ddtrace.bootstrap.sitecustomize in your `app.py` good enough for this to work? Would that be after `gunicorn` imports and patches `gevent` ?

Reply:
Ideally we'd want to run the sitecustomize logic as soon as possible if gevent is being used, so the best thing would be to use `ddtrace-run`. I'm not sure what would happen if we are squeezed in between gunicorn and gevent. Would there be anything preventing a user from using `ddtrace-run` after this change? Note that they could still import from `ddtrace` and perform any configuration they like afterwards.

Reply:
Some users cannot modify the command that is executed to run their app, so they **must** use `from ddtrace import patch_all; patch_all()` for loading/configuring the tracer.

Reply:
In those cases we would have to ensure that the sitecustomize logic is triggered before something like `gevent.monkey.patch_all()` is called. I think there are ways to tell things like `gunicorn` to pre-execute some code? I think we would have to leverage something like that.

Reply:
good to write up the docs now before we release this change :)

Reply:
@P403n1x87 are you talking about adding this sitecustomize logic to gunicorn's `pre_fork()` hook?
For ddtrace/gevent users that don't use gunicorn, does `import ddtrace.bootstrap.sitecustomize` before they call `gevent.monkey.patch_all()` work as a solution?

Reply:
I updated this docstring with the guidance nonusers of ddtrace-run need to follow.